ports/esp8266/gccollect.c

```diff
@@ -26,6 +26,7 @@
 
 #include <stdio.h>
 
+#include "py/mpconfig.h"
```

Comment:
is this needed?

Reply:
Yes, otherwise compiler complain that mp_uint_t isn't defined

Reply:
yes, true!